src/bot.ml

```diff
@@ -28,12 +28,23 @@ let bot_info : Bot_components.Bot_info.t =
   ; email= Config.bot_email toml_data
   ; domain= Config.bot_domain toml_data }
 
+let key = Config.github_private_key
+
+let app_id = Config.github_app_id toml_data
+
 let github_mapping, gitlab_mapping = Config.make_mappings_table toml_data
 
 let github_of_gitlab = Hashtbl.find gitlab_mapping
 
 let gitlab_of_github = Hashtbl.find github_mapping
 
+let installation_tokens : (string, string * float) Base.Hashtbl.t =
+  Hashtbl.create (module String)
```

Comment:
Wouldn't it make sense to move this definition to `Helpers` (or an entirely new module exporting only the `action_as_github_app` function)? This way, it wouldn't be necessary to pass this variable as a parameter at every `action_as_github_app` call.